python/paddle/fluid/dygraph/jit.py

```diff
@@ -506,7 +507,7 @@ def _build_load_path_and_config(path, config):
 @switch_to_static_graph
 def save(layer, path, input_spec=None, **configs):
     """
-    Saves input Layer as ``paddle.jit.TranslatedLayer``
+    Saves input Layer or function as ``paddle.jit.TranslatedLayer``
```

Comment:
Since function cannot save parameters, you should give a Note or Warning and had better explain why for users.

Reply:
Done, thx.